src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.ConvertServiceImpl;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriteServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT = "src/main/java/resources/input file.csv";
+    private static final String REPORT = "src/main/java/resources/report file.csv";

```

Comment:
Try not to use whitespaces in your development file names.
```suggestion
    private static final String INPUT = "src/main/java/resources/input_file.csv";
    private static final String REPORT = "src/main/java/resources/report_file.csv";

```